streaming/base/dataset.py

```diff
@@ -157,8 +158,9 @@ class StreamingDataset(IterableDataset):
             partitioned over the workers. Defaults to ``None``.
         shuffle (bool): Whether to iterate over the samples in randomized order. Defaults to
             ``False``.
-        shuffle_algo (str): Which shuffling algorithm to use. Defaults to ``py1s``.
+        shuffle_algo (str): Which shuffling algorithm to use. Defaults to ``py1b``.
```

Comment:
Can you add a one to line line of brief statement explaining when to use which `naive, py1b, py1s, py2s` algorithm ?